model-optimizer/extensions/back/SpecialNodesFinalization.py

```diff
@@ -125,6 +125,10 @@ class RemoveConstToResult(BackReplacementPattern):
     """
     enabled = True
     force_clean_up = True
+    # TODO: remove this transformation once all plugins support constant value network.
+    # Now it avoids to be run recursively since Const->Result sub-graph can be encountered in a body graph of Loop node
```

Comment:
nit: change to `Do not run recursively since Const->Result sub-graph can be encountered in a body graph of Loop node`. And also explain what this sub-graph is responsible for.